tests/common/snappi/common_helpers.py

```diff
@@ -550,89 +617,124 @@ def get_pfcwd_config_attr(host_ans, config_scope, attr):
     return None
 
 
-def get_pfcwd_poll_interval(host_ans):
+def get_pfcwd_poll_interval(host_ans, asic_value='None'):
     """
     Get PFC watchdog polling interval
     Args:
         host_ans: Ansible host instance of the device
+        asic_value: asic value of the host
+
     Returns:
         Polling interval in ms (int) or None
     """
-    val = get_pfcwd_config_attr(host_ans=host_ans,
-                                config_scope='GLOBAL',
-                                attr='POLL_INTERVAL')
+    if asic_value == 'None':
+        val = get_pfcwd_config_attr(host_ans=host_ans,
+                                    config_scope='GLOBAL',
+                                    attr='POLL_INTERVAL')
+    else:
+        val = get_pfcwd_config_attr(host_ans=host_ans,
+                                    config_scope='GLOBAL',
+                                    attr='POLL_INTERVAL',
+                                    namespace=asic_value)
 
     if val is not None:
         return int(val)
 
     return None
 
 
-def get_pfcwd_detect_time(host_ans, intf):
+def get_pfcwd_detect_time(host_ans, intf, asic_value='None'):
     """
     Get PFC watchdog detection time of a given interface
     Args:
         host_ans: Ansible host instance of the device
         intf (str): interface name
+        asic_value: asic value of the host
+
     Returns:
         Detection time in ms (int) or None
     """
-    val = get_pfcwd_config_attr(host_ans=host_ans,
-                                config_scope=intf,
-                                attr='detection_time')
+    if asic_value == 'None':
+        val = get_pfcwd_config_attr(host_ans=host_ans,
+                                    config_scope=intf,
+                                    attr='detection_time')
+    else:
+        val = get_pfcwd_config_attr(host_ans=host_ans,
+                                    config_scope=intf,
+                                    attr='detection_time',
+                                    namespace=asic_value)
 
     if val is not None:
         return int(val)
 
     return None
 
 
-def get_pfcwd_restore_time(host_ans, intf):
+def get_pfcwd_restore_time(host_ans, intf, asic_value='None'):
```

Comment:
asic_value=None

Reply:
its actually 'None' because get_multidut_snappi_ports appends asic_value to 'None'